org/teams.yaml

```diff
@@ -0,0 +1,429 @@
+          - JimmyCYJ
+          - john-a-joyce
+          - kyessenov
+          - lambdai
+          - linsun
+          - lizan
+          - Nino-K
+          - nmittler
+          - nrjpoddar
+          - PiotrSikora
+          - ramaraochavali
+          - rlenglet
+          - rshriram
+          - sbezverk
+          - tiswanso
+          - venilnoronha
+          - yxue
+        teams:
+          WG - Networking Maintainers/Data Plane:
+            description: Maintainers of data plane components
+            members:
+              - bianpengyuan
+              - duderino
+              - gargnupur
+              - JimmyCYJ
+              - kyessenov
+              - lambdai
+              - linsun
+              - lizan
+              - PiotrSikora
+              - rshriram
+              - venilnoronha
+              - yxue
+          WG - Networking Maintainers/Pilot:
+            description: Maintainers of particular parts of Pilot.
+            members:
+              - costinm
+              - howardjohn
+              - hzxuzhonghu
+              - ramaraochavali
+              - rshriram
+      WG - Perf and Scalability  Maintainers:
+        description: Maintainers for the Perf and Scalability working group.
+        members:
+          - mandarjog
+          - suryadu
+      WG - Policies and Telemetry  Maintainers:
+        description: Maintainers for the Policy and Telemetry working group.
+        members:
+          - bianpengyuan
+          - douglas-reid
+          - gargnupur
+          - kyessenov
+          - mandarjog
+          - nrjpoddar
+      WG - Security Maintainers:
+        description: Maintainers for the Security working group
+        members:
+          - incfly
+          - JimmyCYJ
+          - lei-tang
+          - liminw
+          - lizan
+          - myidpt
+          - yangminzhu
+      WG - Test and Release Maintainers:
+        description: Maintainers for the Test and Release working group.
+        members:
+          - brian-avery
+          - carolynhu
+          - duderino
+          - ericvn
+          - fejta
+          - fpesce
+          - howardjohn
+          - jwendell
+          - linsun
+          - mandarjog
+          - nmittler
+          - richardwxn
+          - sdake
+          - stevenctl
+          - therealmitchconnors
+      WG - User Experience Maintainers:
+        description: Maintainers for the User Experience working group.
+        members:
+          - esnible
+          - irisdingbj
+          - jasonwzm
+          - liamawhite
+          - nmittler
+          - therealmitchconnors
+  Release Managers:
+    description: Current set of release managers.
+    members:
+      - brian-avery
+      - dgn
+      - ericvn
+      - fpesce
+      - howardjohn
+      - irisdingbj
+      - jacob-delgado
+      - JimmyCYJ
+      - johnma14
+      - jwendell
+      - rlenglet
+      - sdake
+      - SecurityInsanity
+    teams:
+      Release Managers - 1.3:
```

Comment:
Should we clean these up at some point? Both 1.3 and 1.4 are no longer supported.